docs/datasets/conventions.md

```diff
@@ -89,27 +89,46 @@ attributes are used, they should be covered by the [CF Conventions v1.8].
 ### Spatial Reference
 
 The spatial dimensions of a data variable must be the innermost dimensions, 
-namely `lat`, `lon` for geographic (EPGS:4326, WGS-84) grids, and `y`, `x` 
-for other grids – in this order.
-Datasets that use a geographic (EPGS:4326, WGS-84) grid must provide the 
-1-D coordinates `lat` for dimension `lat` and `lon` for dimension `lon`.
-Datasets that use a non-geographic grid must provide the 1-D coordinates 
-`y` for dimension `y` and `x` for dimension `x`. 
-
-* In case the grid refers to a known spatial reference system (projected CRS), 
-  it must make use of the CRS encoding described in the 
-  [CF Conventions on Grid Mapping], i.e. add a variable `crs`.
-* In case the grid is referring to satellite viewing geometry must provide 
-  2-D coordinates `lat` and `lon` both having the dimensions `y`, `x` – 
-  in exactly this order, and apply the [CF Conventions on 2-D Lat and Lon].
-
-It is expected that the 1-D coordinates have a uniform spacing, i.e. there 
-should be a unique linear mapping between coordinate values and the image grid. 
-Ideally, the spacing should also be the same in x- and y-direction.
-Note, it is always a good practice to add geographic coordinates to 
-non-geographic, projected grids. Therefore, a dataset may also provide the 2-D 
-coordinates `lat` and `lon` in this case. Spatial coordinates must follow 
-the [CF Conventions on Coordinates].
+namely `y`, `x` - in this order. If the coordinate reference system (CRS) 
+is geographic (e.g. CRS is `EPGS:4326`, `WGS84` or `CRS84`) it is ok and common 
+to use dimensions `lat`, `lon` - in this order.
+
+#### Regular grid mappings
+
+If the spatial grid mapping is regular, the spatial dimensions should 
+have corresponding 1-D coordinate variables using the same name. That is, 
+datasets with spatial dimensions `x`, `y` should provide the 1-D coordinate 
+variables `x` and `y`. Spatial coordinates must follow the 
+[CF Conventions on Coordinates].
+
+The 1-D coordinates of regular grids should have a uniform spacing, i.e. there 
+should be a unique linear mapping between coordinate values and the spatial 
+image grid. This implies strictly monotonically increasing or decreasing 
+coordinate values. Ideally, the spacing should also be the same in 
```

Comment:
Not clear what happens if not. I would delete the whole sentence.

Reply:
This doc is not about what happens, if convention is not followed. It is a guide for creating datasets so that interpretation problems are mitigated. 

Let's discuss this another PR, if you insist in deleting the sentence.